media_types/parse_media_type.ts

```diff
@@ -42,7 +42,7 @@ import { consumeMediaParam, decode2331Encoding } from "./_util.ts";
 export function parseMediaType(
   v: string,
 ): [mediaType: string, params: Record<string, string> | undefined] {
-  const [base] = v.split(";");
+  const [base] = v.split(";") as [string];
```

Comment:
A `string.split()` with a non-empty string as a parameter (e.g. `";"`) will always return an array with at least one string, so its safe to continue without runtime checks

Reply:
```ts
"".split(";")
// [ "" ]
"1234".split(";")
// [ "1234" ]
";;".split(";")
// [ "", "", "" ]
```